Automattic/wp-calypso
# Jetpack Manage: Adding close and open tracking events for the single site upsell lightbox

Extends https://github.com/Automattic/jetpack-manage/issues/217

## Proposed Changes

* This PR extends previous work done on the SingleSiteUpsellLightbox component, by making sure all relevant tracks events are firing. This one specifically focusses on tracks event for opening and closing the lightbox.

## Testing Instructions

* Check out this branch, and make sure your partner type is agency.
* Test all three product upsell pages (Backup, Scan, and Search) and make sure the following events are fired:
  * On opening the lightbox: `calypso_jetpack_agency_upsell_lightbox_opened`
  * On clicking either the 'Purchase via Jetpack.com' or 'Issue License' buttons: no close event should fire.
  * On manually closing the lightbox: `calypso_jetpack_agency_upsell_lightbox_closed`

## Pre-merge Checklist
